electron/launcher/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { userActions } from '../../_actions';
+import FormBuilder from '../FormBuilder';
+
+// Future iterations will an action pulled from redux instead of from App
+const Login = props => {
+
+  // Attempts to login automatically from stored session or credentials
+  useEffect(() => {
+    if(props.auth.user.password !== ''){
+      props.login({
+        username: props.auth.user.username,
+        password: props.auth.user.password
+      });
+    }
+  }, []);
+
+  // Error handling 
+  useEffect(() => {
+    if(props.auth.error !== null){
+      let error = JSON.parse(props.auth.error.error);
+      alert(error.message);
+      props.resetError();
+    }
+  }, [props.auth.error]);
+
+  const formFields = [
+    {
+      name: 'username',
+      label: 'Username',
+      type: 'text',
+      required: true
+    },
+    {
+      name: 'password',
+      label: 'Password',
+      type: 'password',
+      required: true
+    }
+  ];
+
+  return (
+    <>
+      <h1>Login Here</h1>
+      <FormBuilder
+        formFields={formFields}
+        submitFunction={props.register}
+        errorMessageGenerator={false}
+        submitText={"Login"}
+      />
+      <p>First time? <Link to="/registration">Register Here!</Link></p>
+    </>
+  );
+};
+
+const mapStateToProps = (state) => {
+  return state;
+}
```

Comment:
perhaps use a common file for `mapStateToProps` to avoid defining multiple times?

Reply:
for this one its actually cause its suppose to be different for each component, but we are currently just returning the entire state object.. For example, we don't have to do
`return state` we should prob be doing `return { stateItemName: state.xxx }`, but this can be changed in the future.

Reply:
Not sure if doing the requested change is recommended. Look at Buraisx's comment, in the future we may want or need to limit the scope of the state that is being mapped to props object.